jenkins/trunk-secondary.yml

```diff
@@ -0,0 +1,90 @@
+- job:
+    name: percona-server-8.0-trunk-secondary
+    project-type: matrix
+    defaults: global
+    description: |
+        Do not change the parameter in this job if you run it manually!
+        Do not edit this job through the web!
+    disabled: false
+    concurrent: false
+    auth-token: pstrunk80
+    node: docker
+    properties:
+    - build-discarder:
+        days-to-keep: -1
+        num-to-keep: 10
+        artifact-days-to-keep: -1
+        artifact-num-to-keep: 10
+    triggers:
+    - reverse:
+        jobs: 'percona-server-8.0-trunk'
+        result: 'unstable'
```

Comment:
What if the result is 'stable'?

Reply:
It means the following  'Trigger even if the build is unstable'.